src/ConversationLearner.ts

```diff
@@ -12,19 +12,26 @@ import { CLDebug } from './CLDebug'
 import { CLClient } from './CLClient'
 import { CLRecognizerResult } from './CLRecognizeResult'
 import { CLModelOptions } from '.'
+import { ILogStorage } from './Memory/ILogStorage'
 
 export class ConversationLearner {
     public static options: CLOptions | null = null
     public static clClient: CLClient
+    public static logStorage: ILogStorage
```

Comment:
should this be static? what if developer wants to have different CL models configured with different ILogStorage instances?

Reply:
Interesting point.   Same could be said for bot storage in general.  Currently they are partitioned by appId.  I'd add this as a nice to have